CRM/Core/BAO/CustomField.php

```diff
@@ -145,7 +137,7 @@ public static function bulkSave($bulkParams, $defaults = []) {
         $logging->fixSchemaDifferencesFor($tableName, ['ADD' => $addedColumns[$tableName]]);
       }
 
-      Civi::service('sql_triggers')->rebuild($params['table_name'], TRUE);
+      Civi::service('sql_triggers')->rebuild($tableName, TRUE);
```

Comment:
@eileenmcnaughton this looks like it was a real bug to me.

Reply:
yes - it would have only updated one - probably in practice that was enough